README.md

```diff
@@ -392,7 +392,7 @@ is of a certain type, like `IsUndefined()`, `IsNull`, `IsNumber` etc.
 It also has useful methods to convert to a Local<T>, for example:
 ```c++
 V8_WARN_UNUSED_RESULT MaybeLocal<Number> ToNumber(Local<Context> context) const;
-V8_WARN_UNUSED_RESULT MaybeLocal<String> ToNumber(Local<String> context) const;
+V8_WARN_UNUSED_RESULT MaybeLocal<String> ToString(Local<Context> context) const;
```

Comment:
The example code is from https://github.com/v8/v8/blob/ee74e718339223345b42935e9f4cbc7be208e557/include/v8-value.h#L355-L361